requests_pkcs12.py

```diff
@@ -77,15 +84,17 @@ def __init__(self, *args, **kwargs):
             raise ValueError('Both arguments "pkcs12_data" and "pkcs12_filename" are missing')
         if pkcs12_data is not None and pkcs12_filename is not None:
             raise ValueError('Argument "pkcs12_data" conflicts with "pkcs12_filename"')
-        if pkcs12_password is None:
-            raise ValueError('Argument "pkcs12_password" is missing')
+        if pkcs12_password != None and pkcs12_password == "":
+            raise ValueError('Password must be 1 or more bytes.')
         if pkcs12_filename is not None:
             with open(pkcs12_filename, 'rb') as pkcs12_file:
                 pkcs12_data = pkcs12_file.read()
+        pkcs12_password_bytes = None
```

Comment:
This and the following assignments lead to misleading error messages in case pkcs12_password is of the wrong type (i.e. neither None nor `str` nor `bytes`).

Proposal for a cleaner solution:

* if None:
  * `pkcs12_password_bytes = None`
* elif str:
  * pkcs12_password_bytes = encode as UTF-8
* elif bytes:
  * Just pass pkcs12_password to pkcs12_password_bytes
* else:
  * Complain about invalid type of pkcs12_password, explaining that None, str or bytes is required.